modules/ztp-hub-factory-pipeline.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="hub-factory-pipeline_{context}"]
+= The hub factory pipeline
+
+The hub configuration pipeline stage prepares the hub cluster to deploy multiple edge clusters for the end customer.
+
+.Prerequisites
+
+* An {product-title} hub cluster available with {product-version} installed.
+* The correct DNS entries and the API and ingress exposed in the factory network (DHCP external network).
+* At least 6 persistent volumes available and ready to be bound.
+* Internet connectivity.
+* OpenShift Pipelines already created.
+
+
+The flow associated with deploying the hub cluster is:
+
+Check hub::
+
+The initial stages in the hub pipeline downloads the various tools needed. It downloads `jq`, `oc`, `opm` and `kubectl`. It also proceeds to verify that various hub install prerequisites exist before proceeding, for example it checks the:
+
+* {product-title} version.
+* Nodes are ready.
+* Cluster Operators are ready.
+* Metal^3^ pods are ready.
+* Persistent volumes are created.
+* DNS requirements are satisfied.
+
+Deploy HTTPD::
+
+This step deploys and configures an HTTP server on the hub cluster. It obtains the Red Hat Enterprise Linux CoreOS (RHCOS) ISO and RootFS images from link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/[mirror.openshift.com] and ensures these are hosted on the deployed HTTPD server. These are then available to install on the edge cluster.
+
+Deploy registry::
+
+This step deploys a registry on the hub cluster. The substeps involved in this process are:
+
+   * Deploy the registry on the hub.
+   * Sync the {product-title} and Operator Lifecycle Manager (OLM) images from Quay and RedHat registries to the internal registry.
+   * Update the pull secret globally.
+   * Take a snapshot of the created mirror registry.
```

Comment:
I don't see where we do this. Need to clarify if/how this step is done.

Reply:
OK incorrect line - thanks